src/keycloak/admin/userroles.py

```diff
@@ -0,0 +1,41 @@
+import json
+
+from keycloak.admin import KeycloakAdminBase
+
+__all__ = ('UserRoles',)
+
+
+class UserRoles(KeycloakAdminBase):
```

Comment:
To stay in line of the architecture of the client I suggest the following:

Create a class called `UserRoleMapping` which is actually just an empty class (for now) with only one property called `realm` which returns an instance of `UserRoleMappingRealm`. This class has two methods `available` and `add`.

Something like this:

```
class UserRoleMappings(KeycloakAdminBase):

    def __init__(self, realm_name, user_id, *args, **kwargs):
        self._realm_name = realm_name
        self._user_id = user_id
        super(UserRoleMappings, self).__init__(*args, **kwargs)

    @property
    def realm(self):
        return UserRoleMappingsRealm(
            realm_name=self._realm_name,
            user_id=self._user_id,
            client=self._client
        )


class UserRoleMappingsRealm(KeycloakAdminBase):
    _paths = {
        'available': '/auth/admin/realms/{realm}/users/{id}' +
                          '/role-mappings/realm/available',
        'single': '/auth/admin/realms/{realm}/users/{id}' +
                     '/role-mappings/realm'
    }

    def __init__(self, realm_name, user_id, *args, **kwargs):
        self._realm_name = realm_name
        self._user_id = user_id
        super(UserRoleMappingsRealm, self).__init__(*args, **kwargs)

    def available(self):
        return self._client.get(
            url=self._client.get_full_url(
                self.get_path(
                    'available', realm=self._realm_name, id=self._user_id
                )
            )
        )

    def add(self, roles):
        """
        :param roles: _rolerepresentation keycloak api
        """
        return self._client.post(
            url=self._client.get_full_url(
                self.get_path(
                    'single', realm=self._realm_name, id=self._user_id
                )
            ),
            data=json.dumps(roles, sort_keys=True)
        )
```
When the property on the `User` class is updated it can be used on the following way:

```
client = KeycloakAdmin(*kwargs)
client.realms.by_name('my-realm').users.by_id(123).role_mappings.realm.available()
client.realms.by_name('my-realm').users.by_id(123).role_mappings.realm.add(roles)
```
Maybe it's also friendly for future users to give an example of the contents of the roles argument of the `add`-method.